plugins/Actions/ArchivingHelper.php

```diff
@@ -424,19 +290,17 @@ private static function updateActionsTableRowWithGoals(array $row, bool $isPages
 
                 // Calculate revenue per entry
                 if (isset($row[PiwikMetrics::INDEX_GOAL_REVENUE_ENTRY])) {
-                    $row[PiwikMetrics::INDEX_GOAL_REVENUE_PER_ENTRY] = Piwik::getQuotientSafe(
+                    $row[PiwikMetrics::INDEX_GOAL_REVENUE_PER_ENTRY] = (float) Piwik::getQuotientSafe(
```

Comment:
The cast was probably added to get consistency for revenue values in tests when the value is zero. A float will show as `0.00` whereas an int will show `0`, `getQuotientSafe()` returns a float value if it can do the division, otherwise it returns an int. It might be better if it always returned a float value?

Reply:
Ah ok. That makes sense. Even though I guess we have that quite inconsistent across the code base.